### PR TITLE
GrowthCode RTD  : initial release

### PR DESCRIPTION
**Pull request by southern-growthcode**
## Type of change

- [x] New bidder adapter   
- [x] Other

## Description of change

New Growthcode RTD module for delivering real-time contextual and audience segmentation capabilities.

Contact: chirs@growthcode.io

### test parameters for validating bids:
```
pbjs.setConfig(
    ...
    realTimeData: {
         auctionDelay: 1000,
          dataProviders: [
          {
            name: 'growthCodeRtd',
            waitForIt: true,
            params: {
              pid: 'TEST01',
            }
          }
       ]
    }
    ...
}
```
### Testing
To view an example of GrowthCode backends:

`gulp serve --modules=userId,rtdModule,appnexusBidAdapter,growthCodeRtdProvider,sharedIdSystem,criteoBidAdapter`

and then point your browser at:

`http://localhost:9999/integrationExamples/gpt/growthcode.html`

## Other information
prebid.github.io update is in progress.

### CODE REVIEW

**Comment by southern-growthcode** [comment]
Docs Done and approved:  https://github.com/prebid/prebid.github.io/pull/4519

File: modules/growthCodeRtdProvider.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+pbjs.setConfig(
+    ...
+    realTimeData: {
+         auctionDelay: 1000,
+          dataProviders: [
+          {
+            name: 'growthCodeRtd',
+            waitForIt: true,
+            params: {
+              pid: 'TEST01',
```
Comment (ncolletti) [inline modules/growthCodeRtdProvider.md:28]: This pid is returning an empty response. Could this be updated to return a test response to confirm your rtd functionality in the auction?

File: modules/growthCodeRtdProvider.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+## GrowthCode Real-time Data Submodule
```
Comment (ncolletti) [inline modules/growthCodeRtdProvider.md:1]: Please include a section explaining what is adjusting within the bidrequest and what targeting data will be added

**Review by ncolletti — COMMENTED** [review]
The rtd module code looks good. minor adjustments needed to your md file

File: modules/growthCodeRtdProvider.md (hunk 1 above)
Comment (southern-growthcode) [reply modules/growthCodeRtdProvider.md:28]: Server also now returns a response

Comment (ncolletti) [reply modules/growthCodeRtdProvider.md:28]: I'm seeing empty responses from the rtd endpoint and errors in the console from your id module when viewing growthcode.html integration example. Can you resolve these and make sure the rtd module returns example data in the response with this pid? Thank you. We will want to confirm your rtd module behavior in this page.

**Review by ncolletti — APPROVED** [review]
looks good

**Pull request merged**